package.json

```diff
@@ -59,6 +59,7 @@
     "react": "^15.3.2",
     "react-addons-create-fragment": "^15.3.2",
     "react-addons-pure-render-mixin": "^15.3.2",
+    "react-addons-shallow-compare": "^15.4.2",
```

Comment:
react-addons-shallow-compare was added because we could not run reaction without installing it.